cove_ocds/tests.py

```diff
@@ -620,3 +620,28 @@ def test_schema_ocds_extended_release_schema_file():
     assert not schema.extended
     assert not schema.extended_schema_file
     assert not schema.extended_schema_url
+
+
+@pytest.mark.django_db
+def test_schema_after_version_change(client):
```

Comment:
These tests look good, but they only test the second bullet point (not the first) on https://github.com/OpenDataServices/cove/issues/622